Jenkinsfile

```diff
@@ -212,6 +218,42 @@ pipeline {
                 }
             }
         }
+
+        stage('7. Push Docker Images To GoogleCloud') {
+            when {
+                expression {
+                    return isReleaseBranch() || params.PUSH_TO_VORVAN
+                }
+            }
+            agent {
+                docker {
+                    registryCredentialsId HARBOR_CREDS
+                    registryUrl HARBOR_URL
+                    image JAVA_IMAGE
+                    label NODE_LABEL
+                }
+            }
+            steps {
+                timeout(time: 30, unit: 'MINUTES') {
+                    script {
+                        def gitCommitHash = env.GIT_COMMIT
+                        def imageTags = "${versionText},${gitCommitHash}"
+                        withDockerCredentials(DOCKERHUB_CREDS, "FROM_") {
+                            withGCRCredentials(VORVAN_CRED) {
+                                def gcrCreds = readFile("${GCR_JSON_KEY}")
+                                withEnv(['TO_DOCKER_USERNAME=_json_key', "TO_DOCKER_PASSWORD=${gcrCreds}"]) {
```

Comment:
just in case you'd like to do it consistently as jib flags, not to have some things taken from env and some from flags, a version with only flags used can be found here: https://github.com/h2oai/h2oai-monitor/blob/master/Jenkinsfile#L129-L137

Reply:
does it show in jenkins console logs? I thought it did so I used env vars

Reply:
I did as you suggested but had to revert because of errors, in order to debug error will have to turn on console logs, so I think best to have it with env vars.